packages/espressocash_app/lib/features/kyc_sharing/services/kyc_service.dart

```diff
@@ -0,0 +1,226 @@
+// ignore_for_file: avoid_positional_boolean_parameters
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dfunc/dfunc.dart';
+import 'package:flutter/foundation.dart';
+import 'package:injectable/injectable.dart';
+import 'package:kyc_client_dart/kyc_client_dart.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../../../di.dart';
+import '../../accounts/auth_scope.dart';
+import '../../feature_flags/services/feature_flags_manager.dart';
+import '../data/kyc_repository.dart';
+import '../models/document_type.dart';
+import '../utils/kyc_exception.dart';
+import '../utils/kyc_utils.dart';
+
+// Hardcoded for now
+const partnerAuthPk = 'J4Bi8wQnvcX4kLyiA7xemJ7t4bikDncgWUZAscvymGPq';
```

Comment:
This key should be coming from the partner

Reply:
Hmmm, yeah currently we grant partner access on basic info screen. I think in future this should give access to all available partners or only 1 specific?

[figma](https://www.figma.com/design/DtxaVafPEMzSqpdMWorQMB/Design-of-Espresso-Cash-App?node-id=17830-14640&node-type=frame&t=rOhVkcCs0eSZ3OEZ-0)

Reply:
And also, this screen can be opened from profile page, which won't be linked to any partner